src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    enum Color {
+        RED,
+        GREEN,
+        BLUE,
+        WHITE
+    }
+
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        int numberOfColors = Color.values().length;
+        int index = random.nextInt(numberOfColors);
```

Comment:
You can reduce number of variables 
```suggestion
        int index = random.nextInt(Color.values().length);
```